packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -646,43 +675,157 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
     return globalOffset & object.semanticBounds.size;
   }
 
-  /// Removes focus from a node that has the primary focus, and cancels any
-  /// outstanding requests to focus it.
+  /// Moves the primary focus to another node when this one is done with it.
```

Comment:
This line is explaining the side affect of unfocus, we should put it below line 682.